terraform/ses.tf

```diff
@@ -0,0 +1,48 @@
+resource "aws_iam_user_policy" "gitlab-ses-email" {
+  name = "${var.cluster_name}-gitlab-ses-email"
+  user = aws_iam_user.gitlab-ses.name
```

Comment:
I'm actually intentionally not using a role here. GitLab wants to send over SMTP, not a native SES call, per https://docs.gitlab.com/omnibus/settings/smtp.html#amazon-simple-email-system-aws-ses - That requires an IAM User, because Users have a `password` attribute that gets converted to SMTP-compatible password.